### PR TITLE
Use Prisma's programmatic API for keystone-next generate

### PR DESCRIPTION
**Pull request by mitchellhamilton**
Same as the previous PRs about using Prisma's programmatic APIs. See the changeset for the options that it accepts.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -12843,13 +12843,6 @@ prism-react-renderer@^1.2.0:
   resolved "https://registry.yarnpkg.com/prism-react-renderer/-/prism-react-renderer-1.2.0.tgz#5ad4f90c3e447069426c8a53a0eafde60909cdf4"
   integrity sha512-GHqzxLYImx1iKN1jJURcuRoA/0ygCcNhfGw1IT8nPIMzarmKQ3Nc+JcG0gi8JXQzuh0C5ShE4npMIoqNin40hg==
 
-prisma@2.18.0:
```
Comment (timleslie) [inline yarn.lock:12846]: Is this right?

Comment (mitchellhamilton) [reply yarn.lock:12846]: Yes, we don't need the Prima CLI anymore

Comment (timleslie) [reply yarn.lock:12846]: Oh, of course, this is what they call the old `@prisma/cli` now 👍

**Review by timleslie — APPROVED** [review]
Looks good 👍

**Pull request merged**